draft-ietf-modpod-group-processes.md

```diff
@@ -272,17 +267,17 @@ Those processes and criteria shall be developed with community input
 and made public, but need not be documented in the RFC series.
 
 The intent of this memo is to provide the widest possible freedom of
-action to the moderators, with a few constraints.  Examples of
-actions that could be taken include:
+action to administrators and moderators, with a few constraints.
+Examples of actions that could be taken include:
 
 - Automated rate limiting mechanisms;
 - Review and approval of submissions/messages;
 - A private or public admonishment;
 - Temporary or permanent bans in one or more fora.
 
 We stress that these are only examples, and not in any way
-prescriptive. The moderator team is free to decide on these or other
-actions.
+prescriptive. administrators and moderators are free to decide on
+these or other actions.
```

Comment:
```suggestion
prescriptive. Administrators and moderators are free to decide on
these or other actions.
```

So how do we get moderation consistency? I assume this RFC will tasks the moderator team to establish and maintain a set of moderation actions together with some guidance on how to apply them, and administrators are expected to follow that guidance? Because otherwise we're back to the status quo, where every forum does what it wants.